docs/reference/changelog-r2023.md

```diff
@@ -8,8 +8,11 @@ Released on ??
     - Added `recognitionColors` field to [SolidBox](../guide/object-solids.md#solidbox) and Tractor PROTO models ([#5606](https://github.com/cyberbotics/webots/pull/5606)).
   - Bug Fixes
     - Fixed Python API `Supervisor.setSimulationMode` which was failing ([#5603](https://github.com/cyberbotics/webots/pull/5603)).
+    - Fixed Python API `Node.enableContactPointsTracking` which was failing ([#5633](https://github.com/cyberbotics/webots/pull/5633)).
     - Fixed Python API `Field.enableSFTracking` and `Field.disableSFTracking` which were failing ([#5640](https://github.com/cyberbotics/webots/pull/5640)).
     - Fixed crash resulting from requesting pose tracking of unsuitable nodes ([5620](https://github.com/cyberbotics/webots/pull/5620)).
+  - Cleanup
+    - Deprecated the C and MATLAB API functions `wb_supervisor_node_enable/disable_contact_point_tracking` in favor of `wb_supervisor_node_enable/disable_contact_points_tracking` to be more consistent with other APIs ([#5633](https://github.com/cyberbotics/webots/pull/5633)).
```

Comment:
I agree with @stefaniapedrazzi. This branch should probably target develop. The original bug fix of the Python API could still be backported to master.

Reply:
> It would be better to move this change/rename to R2023b and not deprecating functions in a bug fix version.

This PR actually fixes a bug in R2023a, so it should aim at the master branch. However, I agree we could move the deprecation warnings to the develop branch to avoid annoying users with this in a revision.

> I also noticed that the `includeDescendants` argument in the disable method doesn't make sense. If it is doesn't match the value given in the enable method the node tracking is not disabled. But there could not be multiple different tracking with or without including the descendants so I don't see the point of passing this argument. So I would remove it to make the API simpler and easier to use.

Good point. I will take care of this.